apps/meteor/app/livechat/server/api/v1/visitor.ts

```diff
@@ -29,14 +30,13 @@ API.v1.addRoute('livechat/visitor', {
 		});
 
 		const { token, customFields } = this.bodyParams.visitor;
-		const guest: ILivechatVisitorDTO = { ...this.bodyParams.visitor };
+		const guest: Parameters<typeof LivechatTyped.registerGuest>[0] = {
```

Comment:
Is this needed?

Reply:
Apparently its not. Not sure Y added it before :thinking: